x/feegrant/filtered_fee_test.go

```diff
@@ -188,3 +189,80 @@ func TestFilteredFeeValidAllow(t *testing.T) {
 		})
 	}
 }
+
+// invalidAllowance does not implement proto.Message
+type invalidAllowance struct {
+}
+
+// compilation time interface implementation check
+var _ feegrant.FeeAllowanceI = (*invalidAllowance)(nil)
+
+func (i invalidAllowance) Accept(ctx sdk.Context, fee sdk.Coins, msgs []sdk.Msg) (remove bool, err error) {
+	return false, nil
+}
+
+func (i invalidAllowance) ValidateBasic() error {
+	return nil
+}
+
+// invalidAllowance2 does not have field
+type invalidAllowance2 struct {
```

Comment:
You may make it to extend `invalidAllowance1`.

Reply:
And I prefer to have specific naming. How about `invalidFieldAllowance`?

Reply:
Is the absence of the field relevant to the failure? It seems like, being `nil` is the trigger, isn't it?

Reply:
Yes, `nil` is the trigger. I fix the test case name.